checkstyle/import-control.xml

```diff
@@ -41,5 +41,7 @@
     <allow pkg="org.gaul.s3proxy" />
     <allow pkg="org.jclouds" />
     <allow pkg="org.glassfish.grizzly" />
+    <allow pkg="com.spotify" />
+    <allow pkg="org.hamcrest.core" />
```

Comment:
What's this dependency?

Reply:
removed. initially it was used for  cases like `assertThat` , later i have resorted to junit and forgot to remove this.